src/helpers.ts

```diff
@@ -318,3 +318,26 @@ export function getLinkedEpics(timelineItems: ZenhubTimelineItem[]): ZenhubIssue
 
     return [...connectedEpics.values()];
 };
+
+/**
+ * Fetches a list of changed files and mark those that contain changes in test files.
+ */
+export async function isPullRequestTested(octokit: OctokitType, pullRequest: PullRequest) {
+    const files = await octokit.rest.pulls.listFiles({
+        owner: ORGANIZATION,
+        repo: pullRequest.base.repo.name,
+        pull_number: pullRequest.number,
+    });
+    const filePaths = files.data.map((file) => file.filename);
+    const testFilePaths = filePaths.filter((filePath) => {
```

Comment:
Especially this is important but I think that this should catch all the test locations we use, right?